src/views/EditorView/Editor/Editor.tsx

```diff
@@ -152,6 +157,24 @@ class Editor extends React.Component<IProps, {}> {
         }
     };
 
+    private getOptionsPanels = () => {
+        if (this.props.activeLabelType !== LabelType.RECTANGLE)
+            return null;
+
+        const editorData: EditorData = EditorActions.getEditorData();
+        return this.props.imageData.labelRects
+            .filter((labelRect: LabelRect) => labelRect.isCreatedByAI && labelRect.status !== LabelStatus.ACCEPTED)
+            .map((labelRect: LabelRect) => {
+                const positionOnImage: IPoint = {x: labelRect.rect.x, y: labelRect.rect.y};
+                const positionOnViewPort: IPoint = RenderEngineUtil.transferPointFromImageToViewPortContent(positionOnImage, editorData);
+                return <LabelControlPanel
+                    position={positionOnViewPort}
```

Comment:
nie krzyczy Ci RCA o to że tutaj brakuje "key"? dodaj mimo wszystko. zawsze to lepiej mieć niż nie mieć